prov/efa/src/efa_av.c

```diff
@@ -327,12 +327,12 @@ int efa_conn_rdm_init(struct efa_av *av, struct efa_conn *conn)
 		 * The shm provider supports FI_AV_USER_ID flag. This flag
 		 * associates a user-assigned identifier with each av entry that is
 		 * returned with any completion entry in place of the AV's address.
-		 * In the fi_av_insert_call below, the &peer->shm_fiaddr is both an input
+		 * In the fi_av_insert call below, the &peer->shm_fiaddr is both an input
```

Comment:
what changed in this line?

Reply:
underscore: `fi_av_insert_call` to `fi_av_insert call`

... Sai had it captured in his PR.